account-kit/smart-contracts/src/ma-v2/client/client.test.ts

```diff
@@ -1,77 +1,70 @@
 import {
-    custom,
-    keccak256,
-    parseEther,
-    publicActions,
-    toHex,
-    type Address,
-  } from "viem";
-  import { privateKeyToAccount } from "viem/accounts";
-  
-  import { LocalAccountSigner, type SmartAccountSigner } from "@aa-sdk/core";
-  
-  import {
-    createSMAV2SignerAccountClient,
-  } from "./client.js";
-  
-  import { local070InstanceOptSep } from "~test/instances.js";
-  import { setBalance } from "viem/actions";
-  import { accounts } from "~test/constants.js";
-  
-  describe("6900 RI installValidation Tests", async () => {
-    const instance = local070InstanceOptSep;
-    const client = instance.getClient().extend(publicActions);
+  custom,
+  keccak256,
```

Comment:
⚠️ **[eslint]** <[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'keccak256' is defined but never used.
<!-- __reviewdog__:ChA0YTllNTUzMWY0OGMzOGNlEgZyZGpzb24= -->